activerecord/test/schema/schema.rb

```diff
@@ -992,16 +992,19 @@
   create_table :records, force: true do |t|
   end
 
-  if supports_foreign_keys?
-    # fk_test_has_fk should be before fk_test_has_pk
-    create_table :fk_test_has_fk, force: true do |t|
-      t.integer :fk_id, null: false
-    end
+  if supports_foreign_keys_in_create?
```

Comment:
I know. You're missing my point. It needs to not do that on SQLite. This code should be capable of running on both backends.